Modules/Loadable/Models/Testing/Cxx/qSlicerModelsModuleWidgetTest1.cxx

```diff
@@ -111,8 +111,6 @@ int qSlicerModelsModuleWidgetTest1( int argc, char * argv[] )
   dynamic_cast<QWidget*>(module.widgetRepresentation())->show();
 
   // Add some more model nodes
-
-  vtkMRMLModelNode* modelNode3 = modelsLogic->AddModel(argv[1]);
```

Comment:
instead of removing the whole line, just remove `vtkMRMLModelNode* modelNode3 = `